packages/sdk/src/store/optimistic.ts

```diff
@@ -0,0 +1,35 @@
+import { Store } from "./base";
+
+const trivial = async () => null;
+
+type Validator<T> = (value: T) => Promise<T | null>;
+
+export const optimistic = <T>(onValidate: Validator<T> = trivial) => {
+  let queue = Promise.resolve();
+
+  return (store: Store<T>) => {
+    store.subscribe((value) => {
```

Comment:
Do we never delete the subscription? Should we expose this option?

Reply:
not really, it's only created once through the lifetime of the program